### ISSUE TITLE
Importing multiple PDFs

### ISSUE THREAD

**Issue by Lean97** [ccbogel/QualCoder#550]
Hi Colin. First of all, thanks for all the effort and work you've been dedicating to QualCoder - it's amazing for us qualitative scholars to have alternatives to proprietary software.

Well, I'm not a coder, so I don't know exactly how you can help solve this issue. But I'm working on a big project, and most of my data is in PDF format. I selected multiple PDF archives from a folder (600+), and although I have downloaded and installed the qpdf, a warning sign pops up saying "Sometimes pdfs are encrypted, download and encrypt using qpdf before trying to load the pdf". Once you click Ok, another warning pops up immediately (and probably other 600+ after this one).

Could you add a checkbox that understands when we hit the "OK" button once, and replicate it for every other identical warning without asking it again?

Thanks,
Leandro.  

ps.: I'm running it on MacOS

**Comment by ccbogel** [comment]
@Lean97 Hi I'm glad you found QualCoder and are giving it a try out.

I have hidden this msg: "Sometimes pdfs are encrypted, download and encrypt using qpdf before trying to load the pdf". Once you click Ok, another warning pops up immediately (and probably other 600+ after this one).

Importing PDFs is a pain - very true. They are not standard text documents (like odt, docx, txt). They are designed for printing, not for importing and editing. You can try using another software or online solution to convert a pdf to plain text. Then import that plain text.

As images: you can use GIMP to import a pdf, each page is a layer. Then save each layer as an image file.

Finally, the remaining solution would be to use a proprietary software that handles pdfs better that QualCoder.

**Comment by glocalglocal** [comment]
How about pdftotext? https://en.wikipedia.org/wiki/Pdftotext

**Comment by Lean97** [comment]
HI @ccbogel , I have created a Quick Action on Automator (MacOS) to convert PDFs to JPEGs or RTFs, so I have converted all the 600+ PDFs to images and am going to code them as images instead. I think will work out this way. The kind of file I'm working on (old newspaper articles) are not suited for text conversion (the resolution is just too low), but thank you and @glocalglocal for taking some time to reply to my question though.
